contentcuration/contentcuration/frontend/channelEdit/views/ImportFromChannels/ChannelList.vue

```diff
@@ -85,6 +85,9 @@
       channelFilter() {
         this.loadPage();
       },
+      languageFilter() {
```

Comment:
Just to check, is this not handled by this? https://github.com/learningequality/studio/pull/2515/files#diff-55045d471e60f20e9b5e371f3c14d3d234c5c504cb3c784b180d354ab6bc6dacR82 Want to make sure there aren't duplicate calls